### PR TITLE
add lambertw recipe from CRAN using the Conda r skeleton helper

### PR DESCRIPTION
**Pull request by Juke34**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/r-lambertw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+test:
+  commands:
+    - $R -e "library('LambertW')"           # [not win]
+    - "\"%R%\" -e \"library('LambertW')\""  # [win]
+
+about:
+  home: https://github.com/gmgeorg/LambertW http://arxiv.org/abs/0912.4554 http://arxiv.org/abs/1010.2265
+    http://arxiv.org/abs/1602.02200
+  license: GPL-2-or-later
```
Comment (dbast) [inline recipes/r-lambertw/meta.yaml:59]:
```suggestion
  license: GPL-2.0-or-later
```

Comment (Juke34) [reply recipes/r-lambertw/meta.yaml:59]: Ok thank you I will have to be more careful with the license

**Comment by Juke34** [comment]
Now lamw recipe has been merged this one might work

**Comment by Juke34** [comment]
Is there a way to rerun the tests without modifying the recipe? I was waiting for r-lawn to be published in conda-forge...

**Comment by dbast** [comment]
Via https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-restart-ci

**Comment by Juke34** [comment]
Any reason why ggplot2 is absent from win and osx architecture?

**Comment by Juke34** [comment]
Ouch actually ggplot2 is not missing, I have a problem more complicated...:
```
ERROR: lazy loading failed for package ‘LambertW’
Error: package or namespace load failed for ‘ggplot2’ in library.dynam(lib, package, package.lib):
 shared object ‘magrittr.dylib’ not found
Error: package ‘ggplot2’ could not be loaded
```

**Comment by dbast** [comment]
Looks like ggplot2 is missing magrittr

**Comment by Juke34** [comment]
I tried adding r-magrittr, but does not work

**Comment by dbast** [comment]
Something is strange ... also this PR is see similar magrittr error https://github.com/conda-forge/staged-recipes/pull/13254 ...

**Comment by dbast** [comment]
stuff got reported and fixed here: https://github.com/conda-forge/r-magrittr-feedstock/issues/8

Retriggering CI.

**Comment by dbast** [comment]
@Juke34 Thanks!

**Pull request merged**